src/main/java/com/bina/varsim/types/variant/Variant.java

```diff
@@ -611,7 +611,14 @@ public VariantType getType(final int ind) {
         we can be assured that correct variant type is
         returned.
          */
-        Alt alt = alts[ind - 1];
+        Alt alt = null;
+        try {
+            alt = alts[ind - 1];
+        } catch (IndexOutOfBoundsException e) {
+            log.error(this.toString());
+            e.printStackTrace();
+            System.exit(1);
```

Comment:
We should not exit here. But instead throw the exception again. As for the stack trace, the JVM would report the stack trace when the exception is handled at the highest level, so we can probably skip it.